packages/@react-types/table/src/index.d.ts

```diff
@@ -63,7 +64,11 @@ export interface SpectrumTableProps<T> extends TableProps<T>, SpectrumSelectionP
    * Handler that is called after a user performs a column resize.
    * Can be used to store the widths of columns for another future session.
    */
-  onResizeEnd?: (widths: Map<Key, ColumnSize>) => void
+  onResizeEnd?: (widths: Map<Key, ColumnSize>) => void,
+  /**
+   * The drag and drop hooks returned by `useDragAndDrop` used to enable drag and drop behavior for the TableView.
+   */
+  dragAndDropHooks?: DragAndDropHooks['dragAndDropHooks']
```

Comment:
TODO: add `@alpha` to let ppl know

Reply:
`@alpha` isn't valid JSDoc and fails lint.... I'll add `@verion alpha`